netstandard/tools/Trimming/Microsoft.NET.Build.Tasks/LockFileCache.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Microsoft.Build.Framework;
+using NuGet.Common;
+using NuGet.ProjectModel;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    internal class LockFileCache
```

Comment:
This is a copy-paste of code from the SDK.  The name of the type matters here since its used to construct a key for lookup.